### ISSUE TITLE
BUG: Shared mailbox iPhone

### ISSUE THREAD

**Issue by MyTheValentinus** [Zimbra-Community/shared-mailbox-toolkit#47]
Hello,

When you have a shared mailbox and you sent a mail from your iPhone for example, the mail go to the sent folder of shared mailbox, not main account.

I have only test on apple devices, probably same problem on other mail client ?

Thanks

**Comment by barrydegraaff** [comment]
yes, to make that happen, you MUST configure the shared account on your iphone as a separate email account. 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0

**Comment by MyTheValentinus** [comment]
I think you don't understand my problem. When i send an email from my primary account, the mail goes to the wrong sent folder of shared mailbox, not in my personal sent folder.

**Comment by MyTheValentinus** [comment]
For information, the sent folder of shared account does not appear on my folder list view on iPhone, i have only Drafts, Inbox and Junk.

**Comment by barrydegraaff** [comment]
That is a feature, not a bug, if you want to disable that, go to preferences->filters->outgoing filters and remove the filter there. 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0

**Comment by MyTheValentinus** [comment]
Even if i disable this filter, the mail go every time to the wrong sent folder. And the shared account Sent folder not appear on iPhone

**Comment by neelabhg** [comment]
Can confirm, this just happened to me as well.

1. I added a new IMAP account to my iPhone, the user email address (user@example)
2. Sent an email from this account (user@example)
3. The sent email shows up in the shared mailbox's (info@example) Sent folder in Zimbra webmail.

**Comment by barrydegraaff** [comment]
In your iphone go to settings > accounts > account > imap > advanced

There you can select the folders where the iphone can put the sent email.

**Comment by neelabhg** [comment]
Thanks for the response. I was about to follow up.

By the way, this issue does not happen with Thunderbird, so at least it's isolated to iOS as far as I know.

Anyway, it does seem like the iPhone is mapping these folders incorrectly (both Sent and Trash).
I had done what you said earlier, but the bad mapping returns after saving it (the change does not save or stick).

I don't know anything about how IMAP works. Does the server specify a mapping from local to server folder?

**Comment by barrydegraaff** [comment]
AFAIK the server does not do anything. As traditionally saving a copy of sent email is a client side thing.

Try removing and re-adding account.

**Comment by neelabhg** [comment]
So I unshared the mailbox. Sent and Deleted, as expected, work correctly.

After that, I shared the mailbox again. After refreshing, the iPhone shows the shared mailbox in the list of folders. Sending an email again puts it in the Sent folder of the shared mailbox.

Updating the mapping from the iOS Settings, like before, does not stick.

I believe this issue may be happening because the iPhone is failing to "see" the Sent and Trash folders of the main account (user@example) -- the account settings does show these folders, but the Mail app does not.
